src/style-value.js

```diff
@@ -16,15 +16,24 @@
 
   function StyleValue() {}
 
-  // TODO: Include parsing logic here.
   StyleValue.parse = function(property, value) {
```

Comment:
How will this work if value should become a sequence? Don't need to address this now, just need add a TODO.